lib/models/UserStudyRole.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const cypher = require('../utils/get-mcrypt-cypher.js')();
+
+function decryptUsername(username) {
+    return cypher.decrypt(new Buffer(username.trim(), 'base64'))
+        .toString().replace(/\0/g, '');
+}
+
+
+module.exports = (bookshelf) => {
+    return bookshelf.extend({
+        tableName: 'casdba.cas_study_user_role_privs',
+        idAttribute: ['username', 'role_id', 'study_id'],
+        role: function() {
+            return this.belongsTo('CasRole', 'role_id');
+        },
+        initialize: function() {
+            this.on('fetched', this.decryptAttributes);
+        },
+
+        decryptAttributes: function() {
```

Comment:
I kinda agree, see below. I think for model usage this is totally cool, though I think the de/encryption bit should still be kept in get-cyper.js